### PR TITLE
XAF RGS Module

### PR DESCRIPTION
**Pull request by andreschenkels**

### CODE REVIEW

**Comment by andreschenkels** [comment]
@ThijsvOers could you do a test?

**Comment by andreschenkels** [comment]
@tarteo is it possible to get a new runboat?

There is a module website_payment_recurring_donations_mollie giving some pre-commit errors on the website in the manifest file.

**Comment by andreschenkels** [comment]
@ThijsvOers this one has the correct runboat you can review this now.

**Review by ThijsvOers — APPROVED** [review]
Checked and was able to generate the xaf file where the RGS reference code was in field Lead Reference

**Review by ByteMeAsap — APPROVED** [review]
LGTM!

File: l10n_nl_rgs_xaf_auditfile_export/models/xaf_auditfile_export.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from odoo import models
+
+
+class XafAuditfileExport(models.Model):
+    _inherit = "xaf.auditfile.export"
+
+    def _get_auditfile_template(self):
+        """return the qweb template to be rendered"""
+        return "l10n_nl_rgs_xaf_auditfile_export.auditfile_template"
```
Comment (tarteo) [inline l10n_nl_rgs_xaf_auditfile_export/models/xaf_auditfile_export.py]: If you remove ```primary="True"``` from the template (`auditfile_template`) we don't need this. Correct me if I'm wrong

Comment (ByteMeAsap) [reply l10n_nl_rgs_xaf_auditfile_export/models/xaf_auditfile_export.py]: @tarteo , you are correct

**Comment by tarteo** [comment]
Ignoring automatic tests results.

**Comment by andreschenkels** [comment]
Hoi @tarteo kun je deze laatste wijziging terugdraaien. Na de laatste wijziging werkt het dus niet meer.

**Pull request merged**